README.md

```diff
@@ -7,6 +7,11 @@
 
 -----
 
+## Installation
+```cmd
```

Comment:
Other ``` blocks have a space before the language, so better to write as ``` cmd.

Reply:
Ha, it interpreted the markdown in my comment. I mean there should be a space after the triple backticks (`), before cmd.

Reply:
Space not mentioned [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)